WalletWasabi.Gui/Converters/ErrorDescriptorsJsonConverter.cs

```diff
@@ -0,0 +1,31 @@
+using Avalonia;
+using Avalonia.Data.Converters;
+using NBitcoin;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using WalletWasabi.Models;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class ErrorDescriptorsJsonConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (value is IEnumerable<Exception> jsonStr)
+			{
+				return JsonConvert.DeserializeObject<ErrorDescriptors>(jsonStr.First().Message);
```

Comment:
Why first? Could be more than 1 error / warning?

Shouldnt it convert them to a multiline string?

Reply:
I think this was guiding his decision: https://github.com/zkSNACKs/WalletWasabi/pull/2204#discussion_r320589018

Reply:
IMO it is because of the Avalonia bug. Currently, @jmacato did a workaround to make it work but he has to follow the "old" solution where an array was passed and only the first element was passed to the View.

Reply:
I covered all my bases and processed all the exceptions instead of processing only the first one :)